app/src/main/java/org/oppia/app/player/exploration/DeafultFontSizeStateListener.kt

```diff
@@ -0,0 +1,8 @@
+package org.oppia.app.player.exploration
+
+import org.oppia.app.model.StoryTextSize
+
+/** Listener for default font size fetch. */
+interface DeafultFontSizeStateListener {
+  fun onDeafultFontSizeLoaded(result: StoryTextSize)
```

Comment:
There is typo error. Please correct it to onDefault for both function and interface name.

Reply:
Done